egs/wsj/s5/steps/nnet3/decode.sh

```diff
@@ -96,11 +96,14 @@ mkdir -p $dir/log
 [[ -d $sdata && $data/feats.scp -ot $sdata ]] || split_data.sh $data $nj || exit 1;
 echo $nj > $dir/num_jobs
 
-
 ## Set up features.
+[ -f $srcdir/online_cmvn ] && online_cmvn=true || online_cmvn=false
```

Comment:
this is a little hard to parse.  Would you mind just making it a regular if-statement?  you can fit it on 2 lines.